src/aap_eda/core/utils/credential_utils.py

```diff
@@ -0,0 +1,154 @@
+#  Copyright 20244 Red Hat, Inc.
```

Comment:
@bzwei 20244 or 2024? also I think credentials.py would be a better name since its already in the utils folder